nifi-provenance-reporting-tasks/pom.xml

```diff
@@ -25,12 +25,14 @@
     <artifactId>nifi-provenance-reporting-tasks</artifactId>
     <packaging>jar</packaging>
 
+
     <dependencies>
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-api</artifactId>
-            <version>1.21.0</version>
+            <version>1.9.2</version>
```

Comment:
be careful, you are downgrading NiFi here (we are currently using version 1.21.0)